### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by AlyAtalla**
final design match Zeplin's design.
 use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @AlyAtalla 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Most of the requirement has been meet. ✅
 
## Required Changes ♻️
- [x] Dear @AlyAtalla, Your over all design is great and you are almost there. The issue I spot on the navbar is that navigation items are `blue` in color while the original item have a silver color with the active item being bold. 

### Your design 
Screen Shot 2023-08-23 at 11 03 46 AM
### Original design 
Screen Shot 2023-08-23 at 11 06 41 AM
- [x] Also the genre on the original design is a bit blurred while the title is fully black, I can see you have used a bold black for both. Also, the space between the genre and the title seems to be a bit wide in your case. Pleae remember we need to deliver the exact copy of the original design 
### Your design 
Screen Shot 2023-08-23 at 11 12 10 AM

### Original design 
Screen Shot 2023-08-23 at 11 10 48 AM

_Check the comments under the review._

## Optional suggestions
- [Optional] I can see some of your changes are already on the base ranch. This is not a good idea as it will make your base branch inconsistent, beside that your code reviewer will not be able to notice all the changes. Please keep every little change in the feature branch for every project you are going to be working on. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/app.scss
Comment (Alusp) [inline src/app.scss:1]: ` a {`
`        text-decoration: none;`
   `   }`

- [ ] Good job so far, in implementing the changes required by the previous reviewer, :clap: however, as mentioned by the previous reviewer, The issue on the navbar is that navigation items are blue in color while the original item have a silver color with the active item being bold. Please resolve that, the possible option would be to use these color code `#121212` for both `Books` and `Categories` and add `opacity: 0.5;` to `Categories` properties.

Below is an illustrations of yours and the given Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

stu


</td>
      <td>

des

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @AlyAtalla, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

close

 ## Highlights

- [ ] Most of the requirement has been met :heavy_check_mark: 
- [ ] Good job overall following the template design :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/app.scss
Comment (AlyAtalla) [reply src/app.scss:1]: thank you

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**